### DISCUSSION TITLE
LineSeries - 2.0.0-RC2 Enumerable vs 2.0.0-RC3.3 ICollection

### DISCUSSION THREAD

**Discussion by henrrymoraes83** [Live-Charts/LiveCharts2#1635] [Q&A] [answered]
Question about Livecharts2 - version 2.0.0-RC3.3 for Avalonia

On 2.0.0-RC2 to change the LineSeries possition I used this command:

```
private LineSeries<ObservablePoint>? _currentLine;
_currentLine.Values = _currentLine.Values.ToArray();
_currentLine.Values.ElementAt(1).X = dataPoint.X;
_currentLine.Values.ElementAt(1).Y = dataPoint.Y;
```

The _currentLine.Values is of type Enumerable

![image](https://github.com/user-attachments/assets/3807dddf-41e0-48a5-89d2-0a6d0b43b07c)

However in RC3.3 it no longer exists, now it has changed to ICollection and I cannot find a way to do the samething.

![image](https://github.com/user-attachments/assets/c2ec78c9-0024-4bd2-8e9e-7d37d577dad4)

Could you help me how to solve this?

**Comment by beto-rodriguez** [comment, marked as answer]
Hi you can use the `CopyTo` method:

```c#
ICollection values = new List<PilotInfo>();
var newValues = new PilotInfo[values.Count];
values.CopyTo(newValues, 0);
```

but in the next version of the library your current code will work, so probably you can just skip this update and wait for `rc4`

**Reply by henrrymoraes83** [reply]
Thank you!!
